bouncer/views.py

```diff
@@ -24,6 +26,7 @@ def annotation(self):
                 doc_type="annotation",
                 id=self.request.matchdict["id"])
         except exceptions.NotFoundError:
+            statsd.incr("views.annotation.404.annotation_not_found")
```

Comment:
No. I suppose having the annotation not found as a separate stat is useful, but perhaps put it in its own hierarchy rather than tying it to a view. Or, if you do want them to be attached to the views then make it easy to aggregate over all the requests a view gets, which you could do by ensuring that all view events are of the form `views.<name>.<status>`. Having an extra part of the hierarchy (like `.annotation_not_found` in this case) makes it tricky to aggregate.